.circleci/config.yml

```diff
@@ -71,11 +71,6 @@ generate-assets-checksums: &generate-assets-checksums
     name: Generate assets precompilation checksum data
     command: git ls-tree -r HEAD $(<.circleci/asset_paths) > tmp/assets_related_checksums
 
-use-example-config-files: &use-example-config-files
-  run:
-    name: Copy example config files into place to be used by tests
-    command: cp config/examples/*.yml config/
```

Comment:
Wait, why do we remove this?

Reply:
Because there's no `config/examples` anymore, this PR removes it and moves its files to `config.`

Reply:
@akostadinov @jlledom if you prefer, I can move them back to `config/examples`, so the only thing the PR will do is get rid of the duplicates in `openshift/system/config` by  unifying the files

Reply:
I prefer returning the examples until we have something to deal with setting credentials in a relatively secure way.